web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -460,11 +440,42 @@ function showHideSidePanel(mode) {
         panel.style.width = '0';
         togBtn.innerText = '>';
         shiftButtonLeft(togBtn);
-        regBtn.style.display = 'none';
+        regBtn.style.display = '';
     }
     // Notify top window of the SidePanel action
     window.top.postMessage(evData, '*');
 }
 
+function showHideSidePanelAdder() {
```

Comment:
I'd wrap both "showHideSidePanel" into a SidePanel class to handle the kind of state which determines which one is shown.

Reply:
still valid